### ISSUE TITLE
Rock Crystal Ore Does Not Generate 1.16.4-1.3.9

### ISSUE THREAD

**Issue by Annimias** [HellFirePvP/AstralSorcery#1669]
I can't seem to find any rock crystal in any of my (new) worlds. Aquamarine ore generates without a problem, even with biomes o' plenty, but there is absolutely no rock crystal ore generation.

The .toml file was not changed at all after installation, and seems to have the same settings that aquamarine has.

I did try to use the fill/replace command in a creative map to see if there would be any changes, but nothing showed up at all in multiple 500x500 areas. Near bedrock, of course.

I don't know if this is a mod issue, or if I just happened to have really bad luck in the sample areas I chose, but if there is a way to fix this issue, please let me know. Thanks.

**Comment by PocketfulOfGeese** [comment]
Modlist? It is an interaction.

**Comment by DiegoBranco** [comment]
I have the same problem. Using the rod also doesn't show any effect. 

Im also not getting any effects from my basic fosic resonator.

Aquamarine is generated just fine.

Modlist: https://pastebin.com/p2KUA82H

**Comment by predundant** [comment]
We are also having this issue on our pack.  We are on 1.16.5, with forge 36.0.14 
Modlist:  https://pastebin.com/0dhWbHJN

It's a long modlist but maybe with the other one you can narrow it down.

**Comment by RainnMannx** [comment]
Here is a list of what is in common.
https://pastebin.com/xPjD9dAt

**Comment by PocketfulOfGeese** [comment]
@predundant have you considered updating Astral Sorcery because you're literally 8 versions behind on a highly experimental release? You're still in the alpha versions.

**Comment by predundant** [comment]
@Doomgull  Fair point, and thanks for mentioning that.  Curseforge and GDLauncher clients both show it as up to date, so we didn't notice we were on an older version.  We'll need to go through our list and check if we have any others stuck on old versions.

**Comment by RainnMannx** [comment]
Updated Astral and checked a fresh world, with the mod list from predundant it does spawn, just crazy rare. Out of around 600 chunks only saw 2 ore. Maybe something is overwriting it? Would it be possible to make the spawn rate configurable? Maybe if the spawn rate could be increased there would be more even if some of it gets blocked or overwritten?

**Comment by PocketfulOfGeese** [comment]
Something is definitely overwriting, because it's very common by default on its own. A configuration to make it even more common wouldn't apply meaningfully here because of that.

**Comment by RainnMannx** [comment]
What is the common amount, tested it with just Astral and it seems to be 1 ore per 5 to 7 chunks.

**Comment by predundant** [comment]
When you did find it, was it in a vanilla biome or a modded one?

**Comment by RainnMannx** [comment]
Would need to recheck it, didn't make note of the biome.
Full pack it's 1 in 300
Just astral alone looks like 1 in 5-7
Took out ext caves and it was 1 in 100
I'll check the biome.
Is there a standard way for other mods to know what not to replace? Or a way to put the gen for astro near the end?

**Comment by PocketfulOfGeese** [comment]
the ore should be spawning in "groups" of a couple per placement. It's not a load-order thing, it's another mod either flooding generation too hard, or overwriting.

**Comment by xaelith** [comment]
Direwolf's pack had a similar issue early on, and I think extcaves turned out to be the culprit, but not 100% sure. One of the darker "stones" found near bedrock (like soapstone, lavastone or similar) was overwriting the rock crystal ore during worldgen.

**Comment by PocketfulOfGeese** [comment]
That can do it, as well. Config option to select what block-types are overwritten hasn't returned yet.

**Comment by RainnMannx** [comment]
Copy, that's why I asked about config for spawning, it would be a round about way to combat it getting overwritten. Would guess the higher the spawn amount the higher the chance more would survive being overwritten. If config is not the way to go then can maybe do custom recipe, or remove a few gen mods. Thanks for your time.

**Comment by PocketfulOfGeese** [comment]
You get into a circular escalation path that way, though, rather than having the problem addressed by the offending mod crowding things out.

That said, yeah, alternative recipes would also work.

**Comment by dukedagmor** [comment]
I'm currently running against this issue whilst playing with friends, after some hours we wanted to start with AstralSorcery only to figure out the modpack "doesn't" generate the ore :(
at this moment we have found 1 ore block after hours of searching
what I find strange is that other ores do generate that low and other modded ores seem not or less affected.
as @xaelith said, extended caves is part of the pack, but then I'd expect the other ore to be overwritten as well unless their is some id conflict.
I guess what I'm really asking for is a possible short term fix so we can get started with the mod whilst keeping our current server running. (other than spawning some in)
